src/Raven.Studio/typescript/common/helpers/database/documentHelpers.ts

```diff
@@ -4,18 +4,18 @@ import document = require("models/database/documents/document");
 
 class documentHelpers {
     static findRelatedDocumentsCandidates(doc: documentBase): string[] {
-        var results: string[] = [];
-        var initialDocumentFields = doc.getDocumentPropertyNames();
-        var documentNodesFlattenedList: any[] = [];
+        const results: string[] = [];
+        const initialDocumentFields = doc.getDocumentPropertyNames();
+        const documentNodesFlattenedList: any[] = [];
 
         // get initial nodes list to work with
         initialDocumentFields.forEach(curField => {
             documentNodesFlattenedList.push(doc[curField]);
         });
 
         for (let documentNodesCursor = 0; documentNodesCursor < documentNodesFlattenedList.length; documentNodesCursor++) {
-            var curField = documentNodesFlattenedList[documentNodesCursor];
-            if (typeof curField === "string" && /\w+\/\w+/ig.test(curField)) {
+            const curField = documentNodesFlattenedList[documentNodesCursor];
+            if (typeof curField === "string" && /\w+\/\w+/ig.test(curField) && curField.length < 512) {
```

Comment:
Can you move the `curField.lenght < 512` before the regex check? That can also improve perf.

Reply:
you are right  - fixing that